examples/e2e_counter/main.go

```diff
@@ -11,7 +11,6 @@ import (
 func main() {
```

Comment:
I think I'll eventually delete this `e2e_counter`.
I feel is quite dirty and these things become a pain to maintain on changes.
Somehow I'd like to include the `go-foldersync` tests in this repo... would make tests much more robust that this `e2e_counter` or something similar.

Two options:
1. Move the go-foldersync just to this repo and make it an example app.
2. Include in the test step of CI, a cycle new extra step of `git clone`, `replace` the dependency, run tests.

Not sure about it.

Reply:
Good call. My vote would be for the first option as it would be easier for other to locally run those tests as well.

Reply:
Like the idea, will make the todo.